windows/windows_update_install/prepare_msu_file.yml

```diff
@@ -24,6 +24,14 @@
 
 - name: "Get unused driver letter"
   include_tasks: ../utils/win_get_unused_drive_letter.yml
+  when: guest_os_build_num | int >= 17763
+
+# For OS older than 17763 (Windows 1809 or Windows Server 2019), it has very high frequency of timeout
+# for executing command to get unused driver letter. Hence here specify one for them.
```

Comment:
driver letter -> drive letter, and also below

Reply:
Fixed. Thanks.